### PR TITLE
Fix the drawing element

### PR DESCRIPTION
**Pull request by printfn**

### CODE REVIEW

**Comment by NickGeek** [comment]
Fixes #190

File: app/src/app/components/note-viewer/elements/drawing/DrawingElementComponent.tsx [hunk 1]
```diff
@@ -7,12 +7,34 @@ import { Input, Row } from 'react-materialize';
 import stringify from 'json-stringify-safe';
 import * as FullScreenService from '../../../../services/FullscreenService';
 
-type Touch = {
+class Touch {
```
Comment (NickGeek) [inline app/src/app/components/note-viewer/elements/drawing/DrawingElementComponent.tsx:10]: Can you please move the auxiliary class definitions to the bottom of the file?

File: app/src/app/components/note-viewer/elements/drawing/DrawingElementComponent.tsx [hunk 2]
```diff
@@ -7,12 +7,34 @@ import { Input, Row } from 'react-materialize';
...
 	identifier: number;
 	x: number;
 	y: number;
+
+	constructor(event: PointerEvent) {
+		this.identifier = event.pointerId;
+		this.x = event.clientX;
+		this.y = event.clientY;
+	}
 };
```
Comment (NickGeek) [inline app/src/app/components/note-viewer/elements/drawing/DrawingElementComponent.tsx:20]: This semi-colon isn't needed

**Review by NickGeek — CHANGES_REQUESTED** [review]

File: app/src/app/components/note-viewer/elements/drawing/DrawingElementComponent.tsx (hunk 1 above)
Comment (printfn) [reply app/src/app/components/note-viewer/elements/drawing/DrawingElementComponent.tsx]: Done

File: app/src/app/components/note-viewer/elements/drawing/DrawingElementComponent.tsx (hunk 2 above)
Comment (printfn) [reply app/src/app/components/note-viewer/elements/drawing/DrawingElementComponent.tsx]: Done

**Review by NickGeek — APPROVED** [review]
LGTM! Thanks.

**Pull request merged**